packages/speech/test/index.js

```diff
@@ -958,10 +958,41 @@ describe('Speech', function() {
         }
       };
 
+      it('should format the results', function(done) {
+        speech.api.Speech = {
+          asyncRecognize: function(reqOpts, callback) {
+            var operation = through.obj();
+            callback(null, operation, apiResponse);
+          }
+        };
+
+        var result = {
+          results: []
+        };
+
+        var formattedResults = [];
+        Speech.formatResults_ = function(results, verboseMode) {
+          assert.strictEqual(results, result.results);
+          assert.strictEqual(verboseMode, false);
+          return formattedResults;
+        };
+
+        speech.startRecognition(FILE, CONFIG, function(err, operation) {
+          assert.ifError(err);
+
+          operation.emit('complete', result, null, null, function(err, resp) {
+            assert.ifError(err);
+            assert.strictEqual(resp, formattedResults);
+            done();
+          });
```

Comment:
@stephenplusplus I'm a bit confused whats going on here. Why is a callback being passed into the emit?

I'm thinking this check would be written as such:
```js
operation.on('complete', function(resp) {
  assert.strictEqual(resp, formattedResults);
  done();
});
operation.on('error', function(err) {
  assert.ifError(err);
});
operation.emit('complete', result, null, null);
```